encrypted-media-respec.html

```diff
@@ -167,7 +167,7 @@
       <p>The following features are <strong>at risk</strong> and may be removed:</p>
       <ul>
         <li><p>The <a def-id="persistent-usage-record-session"></a> session type and the related <a def-id="media-key-session-destroyed-algorithm"></a> algorithm.</p></li>
-        <li><p>Setting the <var>media element</var>'s <a def-id="readystate"></a> value based on key availability in the <a def-id="queue-waitingforkey-algorithm"></a> and <a def-id="resume-playback-algorithm"></a> algorithms.</p></li>
+        <li><p>Setting the <var>media element</var>'s <a def-id="readystate"></a> value based on key availability in the <a def-id="wait-for-key-algorithm"></a> and <a def-id="resume-playback-algorithm"></a> algorithms.</p></li>
```

Comment:
"algorithm**s**" at the end of the sentence covers both. I didn't change anything other than the name.